code/modules/mob/living/simple_animal/abnormality/he/golden_false_apple.dm

```diff
@@ -311,15 +286,29 @@
 	is_maggot = TRUE
 	SpeedChange(-1)
 
-/mob/living/simple_animal/hostile/abnormality/golden_apple/AttackingTarget()
+/mob/living/simple_animal/hostile/abnormality/golden_apple/AttackingTarget()//regular attacks or AOE. Determines the outcome for both players and the AI behavior
 	if(!can_act)
 		return FALSE
-	if(!is_maggot)
+	if(is_maggot == FALSE)//Is it still in the first form? Start building sheen pulses
```

Comment:
Why are you changing `!is_maggot` to `is_maggot == FALSE`, unless you're changing `is_maggot` to have nulls then this is the exact same code exact longer.

Reply:
Taking your advice and changing them not to use TRUE/FALSE when applicable